docs/build/js-client/10-marine-js.md

```diff
@@ -1,63 +1,79 @@
 # Marine JS
 
-# TODO: adapt to JS Client
+## Using Marine services in JS client
 
-## Using Marine services in Fluence JS
+JS client can host Marine services with Marine JS. Currently only pure single-module services are supported.
```

Comment:
I was under impression Valery told me that multi-modules are supported now as well, please make sure this is correct (or is it not supported on js-client side?)

Reply:
Yes, need to support this flow in JS client

Reply:
> Yes, need to support this flow in JS client
when? this looks like as good a time as any.